### ISSUE TITLE
Broken recurring tasks in 1.0.0

### ISSUE THREAD

**Issue by akvarats** [adamcharnock/lightbus#24]
Let consider `bus.py` from https://lightbus.org/latest/howto/schedule-recurring-tasks/

```python
# bus.py
import lightbus

bus = lightbus.create()

@bus.client.every(seconds=1)
def do_it():
    print("Hello!")
```

In venv with `lightbus==1.0.0` nothing happens after `lightbus run`. Also, after terminating the process with Ctrl+C, you see "sys:1: RuntimeWarning: coroutine 'call_every' was never awaited"

For `lightbus==0.10.0` it works fine.

**Comment by adamcharnock** [comment]
Thank you for reporting this @akvarats. I was very surprised to find that this wasn't working, but you're absolutely right that it is a problem.

It seems to be because of some of the more recent internal refactoring I have done. I'm going to look into both an immediate workaround and an actual fix. I'll let you know when I have something.

**Comment by adamcharnock** [comment]
Hi @akvarats,

Ok, a quick workaround is as follows:

```
import lightbus

bus = lightbus.create()

# WORKAROUND: Despite how this reads, this only disables in the internal thread-local bus client creation.
# This may lead it some other issues when accessing the bus from within event handlers (such as a deadlock).
# `bus.client` is a thread-local proxy, and setting `bus.client.enable = false` disables the proxying and only 
# ever creates one client.
bus.client.enabled = False

@bus.client.every(seconds=1)
def do_it():
    print("Hello!")


@bus.client.on_start()
def on_start(client):
    pass
```

**Comment by adamcharnock** [comment]
Hi @akvarats. I think I have a fix, I have released it as `1.0.1a1`. Can you install it and let me know if it resolves your issue?

```
pip install lightbus==1.0.1a1
```

**Comment by akvarats** [comment]
Hi, @adamcharnock. Now it works great, both the workaround and `1.0.1a1`. Thanks!

**Comment by adamcharnock** [comment]
No problem @akvarats, thank you for taking the time to point this out! Let me know if you need any help. My mobile will get pinged if you leave a message in the [discord room](https://discord.gg/2j594ws).

I've now released `1.0.1` stable.
